app/controllers/reports/regions_controller.rb

```diff
@@ -272,22 +280,70 @@ def monthly_state_data_report
     end
   end
 
-  def monthly_district_report
-    return unless current_admin.feature_enabled?(:monthly_district_report)
+  def hypertension_monthly_state_data
+    @medications_dispensation_enabled = current_admin.feature_enabled?(:medications_dispensation)
+    csv = MonthlyStateData::Exporter.new(
+      exporter: MonthlyStateData::Hypertension.new(
+        region: @region,
+        period: @period,
+        medications_dispensation_enabled: @medications_dispensation_enabled
+      )
+    ).report
+    report_date = @period.to_s.downcase
+    filename = "monthly-district-data-#{@region.slug}-#{report_date}.csv"
 
-    @region ||= authorize { current_admin.accessible_district_regions(:view_reports).find_by!(slug: report_params[:id]) }
-    @period = Period.month(params[:period] || Date.current)
-    zip = MonthlyDistrictReport::Exporter.new(@region, @period).export
+    respond_to do |format|
+      format.csv do
+        send_data csv, filename: filename
+      end
+    end
+  end
+
+  def diabetes_monthly_state_data
+    @medications_dispensation_enabled = current_admin.feature_enabled?(:medications_dispensation)
+    csv = MonthlyStateData::Exporter.new(
+      exporter: MonthlyStateData::Diabetes.new(
+        region: @region,
+        period: @period,
+        medications_dispensation_enabled: @medications_dispensation_enabled
+      )
+    ).report
     report_date = @period.to_s.downcase
-    filename = "monthly-district-report-#{@region.slug}-#{report_date}.zip"
+    filename = "monthly-district-data-#{@region.slug}-#{report_date}.csv"
```

Comment:
Maybe we should rename these file names for diabetes and potentially hypertension?

Reply:
yeah, we should